src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgramProperties.java

```diff
@@ -29,6 +29,6 @@
      * @return an example command line for this program
      */
     String usageExample() default "The author of this program hasn't included any example usage, please complain to them.";
-    Class<? extends CommandLineProgramGroup> programGroup() default MiscProgramGroup.class;
+    Class<? extends CommandLineProgramGroup> programGroup() default FastaProgramGroup.class;
```

Comment:
Why did you make FastaProgramGroup the default?  Surely Misc is more appropriate?